lib/Common/Memory/HeapAllocator.h

```diff
@@ -130,6 +130,14 @@ struct HeapAllocator
     static HeapAllocator Instance;
     static HeapAllocator * GetNoMemProtectInstance();
 
+private:
+    HANDLE m_privateHeap;
```

Comment:
It seems that you should initialise this member with nullptr value here or in constructor initializer list.
Otherwise it may contain some random garbage value in Release build and then HeapAllocator::InitPrivateHeap() function will fail to initialize the heap.